content/jmccann/clair/index.md

```diff
@@ -0,0 +1,88 @@
+---
+date: 2017-02-09T00:00:00+00:00
+title: Clair
+author: jmccann
+tags: [ docker, security ]
+repo: jmccann/drone-clair
+logo: clair.svg
+image: jmccann/drone-clair
+---
+
+The Clair plugin submits your docker image to your [Clair](https://github.com/coreos/clair)
+server to scan your docker image for security vulnerabilities.
+
+The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  clair:
+    image: jmccann/drone-clair:1
+    url: http://clair.company.com
+    username: johndoe
+    password: mysecret
+    scan_image: python:2.7
+```
+
+To verify https/ssl connections with a different CA certificate use `ca_cert`
+
+```diff
+pipeline:
+  clair:
+    image: jmccann/drone-clair:1
+    url: http://clair.company.com
+    username: johndoe
+    password: mysecret
+    scan_image: python:2.7
++   ca_cert: |
++     -----BEGIN CERTIFICATE-----
++     MII...
++     -----END CERTIFICATE-----
+```
+
+# Secrets
+
```

Comment:
Sure, I think that makes sense. The only suggest would be taking this sentence:

> The above `username` and `password` Yaml attributes can be replaced with the `DOCKER_USERNAME` and `DOCKER_PASSWORD` secrets.

And maybe making it more generic

> Instead of configuring the username and password in your yaml file in plain text you can use Drone secrets, which substitutes the secrets at runtime. Please see the documentation to learn more.

The only reason for this suggestion is that people could take the current wording too literally and assume the secret names must be DOCKER_USERNAME and DOCKER_PASSWORD, when they can really be any value.

Also note that we don't have to use my suggestion verbatim. It was quickly written and may need to be adjusted for precision :)